apps/frontend/src/app/2_molecules/AssetAmountPriceRenderer/AssetAmountPriceRenderer.tsx

```diff
@@ -10,23 +10,30 @@ type AssetAmountPriceRendererProps = {
   value: Decimalish;
   asset: string;
   className?: string;
+  valueClassName?: string;
+  priceClassName?: string;
 };
 
 export const AssetAmountPriceRenderer: FC<AssetAmountPriceRendererProps> = ({
   value,
   asset,
   className,
+  valueClassName,
+  priceClassName,
 }) => {
   // TODO: get price in usd
   const usdPrice = value;
 
   return (
-    <div className={classNames('flex flex-col text-right', className)}>
-      <AmountRenderer value={value} suffix={asset} />
+    <div className={className ?? 'flex flex-col text-right'}>
```

Comment:
"With this change, you lose the default flex-col text-right classes. Is that intentional?"